src/components/Atoms/Link/Link.js

```diff
@@ -1,47 +1,45 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import { StyledLink, WrapperLink } from './Link.style';
+import StyledLink from './Link.style';
 
-const Link = ({ children, color, href, target, inline, wrapper, ...rest }) => {
+const Link = ({ children, color, href, target, linktype, home, ...rest }) => {
   const window = target === 'blank' ? '_blank' : '_self';
+  const relationship = target === 'blank' ? 'noopener noreferrer' : false;
 
-  return wrapper ? (
-    <WrapperLink {...rest} color={color} inline={inline}>
-      {children}
-    </WrapperLink>
-  ) : (
+  return (
     <StyledLink
       {...rest}
       color={color}
       href={href}
       target={window}
-      inline={inline}
-      as="a"
+      rel={home ? 'home' : relationship}
+      linktype={linktype}
     >
       {children}
     </StyledLink>
   );
 };
 
 Link.propTypes = {
-  /** Inline is link style */
-  inline: PropTypes.bool,
-  wrapper: PropTypes.bool,
-  /** Button color style */
+  /** Link styling: button, standard, etc. See linkStyles in theme. */
+  linktype: PropTypes.string,
```

Comment:
We can define the values `PropTypes.oneOf(['button', 'standard'])`

Reply:
I also would just call it `type`.

Reply:
nice one, will add that